vms/organization/templates/organization/delete.html

```diff
@@ -4,7 +4,7 @@
 
 {% block setting_content %}
     <div class="spacer"></div>
-    <form class="form-horizontal" action="{% url 'organization:delete' organization_id %}" method="post">
+    <form class="form-horizontal" action="" method="post">
```

Comment:
@amruthasangeeth Where is it specified that it is redirecting to OrganizationDeleteView?